src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -481,14 +488,28 @@ export class HuiCardPicker extends LitElement {
           display: block;
           --mdc-shape-small: var(--card-picker-search-shape);
           margin: var(--card-picker-search-margin);
+          position: sticky;
+          top: 0;
+          z-index: 10;
+          background-color: var(
+              --ha-dialog-surface-background,
+              var(--mdc-theme-surface, #fff)
+          );
         }
 
         .cards-container-header {
           font-size: 16px;
           font-weight: 500;
-          padding: 12px 8px 4px 8px;
+          padding: 12px 8px;
           margin: 0;
           grid-column: 1 / -1;
+          position: sticky;
+          top: 56px;
+          z-index: 1;
+          background: linear-gradient(180deg, var(
```

Comment:
```suggestion
          background: linear-gradient(90deg, var(
```

@jpbede what do you think about that? It would make it always readable.

Reply:
It is indeed better readable